website/docs/foundations/accessibility/index.md

```diff
@@ -0,0 +1,61 @@
+
+1. It adds a message to the page to let the screen reader user know that the route has changed and regular page navigation can resume (it is similar to [https://github.com/ember-a11y/a11y-announcer](https://github.com/ember-a11y/a11y-announcer) but does not use `aria-live`).
+2. It moves the focus to that message for the screen reader user, effectively resetting focus in Ember apps (similar to how a native web page/site works).
+3. It provides a bypass mechanism so the user can skip to the page's primary content (see [https://www.w3.org/TR/WCAG20-TECHS/G1.html](https://www.w3.org/TR/WCAG20-TECHS/G1.html)). You can opt out of this if you want (see the `Options` section in the documentation for available options).
+
+## Page Organization
+
+All content should be contained in elements with [landmark roles](https://www.w3.org/TR/wai-aria/#landmark_roles). Typically this will be the `header`, `main`, and `footer` elements, but depending on your page layout, you might also be using `aside` and `nav` as direct descendants of the `body` element as well.
+
+## Titles and Labels
+
+### Page titles
+
+Ensure that each page has a relevant page title. This is included in Ember applications by default, but older applications that have been updated over the years may need to manually add this in with [ember-page-title](https://github.com/ember-cli/ember-page-title).
+
+### Headings
+
+Headings (h1-h6) should follow a logical order and not skip any levels. This can be tricky in dynamic applications so watch out for this one.
+
+### Input labels
+
+All inputs should have associated labels. These are built-in to the form components. If you need to compose custom fields using the base elements, remember that a `placeholder` attribute is not an acceptable substitute.
+
+### Accessible name
+
+Not every element is allowed to use `aria-label`. Check the [element's role documentation](https://www.w3.org/TR/wai-aria/#role_definitions) to see what aria-attributes are permissible. When in doubt, ask in [#team-design-system](https://hashicorp.slack.com/archives/C7KTUHNUS)! As the saying goes, "no ARIA is better than bad ARIA!"
+
+## Focus Management
+
+There are some key points to remember about focus management:
+
+1. Focus should return from whence it came. If a button toggled a modal window, closing that modal window should return focus to the button.
+2. Trap the focus! Focus should remain in a modal or other popup when it is open. Try [ember-focus-trap](https://github.com/josemarluedke/ember-focus-trap).
+3. Focus order should match DOM order.
+
+## Automated Testing
+
+Use available automated tooling for static and dynamic analysis. Ember apps can use [ember-template-lint](https://github.com/ember-template-lint/ember-template-lint) for static analysis and [ember-a11y-testing](https://github.com/ember-a11y/ember-a11y-testing) for dynamic analysis. Additionally, consider adding HTML validation to your CI.
+
```

Comment:
Just a note: If we are recommending others consider validating their HTML then we may want to look at making sure our components use valid markup as well. (For example currently we use div tags inside our button component which is invalid.)

Reply:
I agree! I think we have an issue in our backlog.